config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   root 'users#index'
   
   resources :users, only: [:index, :show] do
```

Comment:
- [ ] It's about the route on the login wireframe: the wireframe recommends accessing the login at `.../login` but it pops No Route. I had to navigate to `.../users/sign_in` before the login views could be displayed.
- Ideally, until a user is signed in, he should have access to the user's route Kindly fix this to appropriately match the requirement - user story for ease of navigation and protection of the integrity of your user's info.